### PR TITLE
[doc][minor] pytorch 1.12 + conda

### PR DESCRIPTION
**Pull request by blefaudeux**
## What does this PR do?
- pytorch nightly is not required anymore, since pytorch 1.12 is out
- adding a minimal conda env which handles the cudatoolkit dependencies out of the box
- minor matplotlib upgradefix

## Before submitting

- [ ] Did you have fun?
  - Make sure you had fun coding 🙃
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/xformers/blob/master/CONTRIBUTING.md)?
- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
  - [ ] N/A
- [ ] Did you make sure to update the docs?
  - [ ] N/A
- [ ] Did you write any new necessary tests?
  - [ ] N/A
- [ ] Did you update the [changelog](https://github.com/facebookresearch/xformers/blob/master/CHANGELOG.md)? (if needed)
  - [ ] N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

File: xformers/benchmarks/utils.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def pretty_print(results, title, units):
 
 
 def pretty_plot(
-    results, title, units: str, filename=None, dash_key="", legend_loc="bottom_right"
+    results, title, units: str, filename=None, dash_key="", legend_loc="lower right"
```
Comment (blefaudeux) [inline xformers/benchmarks/utils.py:59]: this changed with a recent matplotlib

File: requirements-test.txt [hunk 2]
```diff
@@ -27,4 +27,4 @@ hydra-core >= 1.1
 fairscale >= 0.4.5
 
 # Dependency for fused layers, optional
-triton == 2.0.0.dev20220403
+triton == 2.0.0.dev20220701
```
Comment (blefaudeux) [inline requirements-test.txt:30]: updating triton while I'm at it, newer versions include bugfixes

**Comment by blefaudeux** [comment]
landing since really minor, I hope that works @fmassa and @dianaml0 (please tell me if you prefer that I don't do that ever, I won't be offended I promise)

**Comment by blefaudeux** [comment]
adding @Vittorio-Caggiano in case you have an opinion, I figured that providing a ready-to-use conda config was nice to have and I know you were interested in that at some point

**Comment by Vittorio-Caggiano** [comment]
yes, I think it is a good idea!

**Pull request merged**